ra-tls/elv/ra-tls-client.c

```diff
@@ -111,38 +110,34 @@ int ra_tls_echo(int sockfd)
 
 	int derSz;
 	const unsigned char *der = wolfSSL_X509_get_der(srvcrt, &derSz);
-
-	sgx_quote_t quote;
-	get_quote_from_cert(der, derSz, &quote);
-	sgx_report_body_t *body = &quote.report_body;
-
-	printf("Server's SGX identity:\n");
-	printf("  . MRENCLAVE = ");
-	for (int i = 0; i < SGX_HASH_SIZE; ++i)
-		printf("%02x", body->mr_enclave.m[i]);
-	printf("\n");
-
-	printf("  . MRSIGNER  = ");
-	for (int i = 0; i < SGX_HASH_SIZE; ++i)
-		printf("%02x", body->mr_signer.m[i]);
-	printf("\n");
+        sgx_report_body_t *body = NULL;
+        uint8_t quote_buff[8192] = {0,};
+        get_quote_from_cert(der, derSz, (sgx_quote_t*)quote_buff);
+        sgx_quote_t* quote = (sgx_quote_t*)quote_buff;
+        body = &quote->report_body;
+        printf("Server's SGX identity:\n");
+        printf("  . MRENCLAVE = ");
+        for (int i = 0; i < SGX_HASH_SIZE; ++i)
+                printf("%02x", body->mr_enclave.m[i]);
+        printf("\n");
+        printf("  . MRSIGNER  = ");
+        for (int i = 0; i < SGX_HASH_SIZE; ++i)
+                printf("%02x", body->mr_signer.m[i]);
+        printf("\n");
 
```

Comment:
Please fix the code format.

Reply:
Do you mean the body of for should be blcoked by "{}" ?

Reply:
Done.